pkg/clients/aws/route53_test.go

```diff
@@ -99,6 +101,13 @@ var awsSecret = &v1.Secret{
 	},
 }
 
+var testClusterDeployment = &hivev1.ClusterDeployment{
```

Comment:
Can we create a unit test where the cluster deployment is in STS mode to test the new code?

Reply:
:+1: